models/jira.yml

```diff
@@ -28,8 +28,60 @@ models:
         description: The ID of the status that an issue has on a given day.
       - name: status
         description: The name of the status that an issue has on a given day.
+      - name: story_points
+        description: The story points that an issue has on a given day. 
+      - name: story_point_estimate
+        description: The story point estimate that an issue has on a given day.  
```

Comment:
These fields are also now included within the `jira__issue_enhanced` model by default as well. Be sure to add the docs for those fields in the appropriate spot.